packages/server/src/api/controllers/row/index.ts

```diff
@@ -103,7 +105,22 @@ export async function destroy(ctx: any) {
   const inputs = ctx.request.body
   const tableId = utils.getTableId(ctx)
   let response, row
+
   if (inputs.rows) {
+    const targetRows = inputs.rows.map(
+      (row: { [key: string]: string | string }) => {
+        let processedRow = typeof row == "string" ? { _id: row } : row
```

Comment:
I think some type definitions can help a lot here, as the exact expected format is quite hard to understand declaratively. 

i.e. we should add a DeleteRowsRequest api type that expresses the various forms described in the pr description. That can be integrated into the signature like ctx: Ctx<DeleteRowsRequest>. The ctx.body will then automatically infer that type and the usages in the controller will become type safe